webcomponents.js/webcomponents.js-tests.ts

```diff
@@ -10,10 +10,12 @@ var fooProto = Object.create(HTMLElement.prototype, {
     }
 });
 
-document.registerElement("x-foo", {
+var XFoo = document.registerElement("x-foo", {
     prototype: fooProto
 });
 
+var xFoo = new XFoo();
+
```

Comment:
Thanks lgrignon. I am author of new definitions.

> I provided AbstractPolymerElement in tests as a convenience for developer who wish to integrate easily, maybe you could provide an AbstractDomModule, and an additional test case to register an instance of a simple implementation, showing the object-oriented way of using Polymer ?

Sorry, I think that polymer.d.ts should not supply utilities but only type definitions. In the tests, the implementations are conforming to Polymer developer guides.

> I started to write the definitions of the core (now Iron) & paper components' library. It would be nice to write at least the one I wrote for 0.5 (except the obsoletes one, only a few)

The Repositories of iron-elements and paper-elements (and more elements) are separated:   [PolymerElements/iron-elements](https://github.com/PolymerElements/iron-elements) and [PolymerElements/paper-elements](https://github.com/PolymerElements/paper-elements).
 If repositories are different, I think we should create different definitions.

> I can help if you want :)

Thank you!